### PR TITLE
Authentication page

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Added sign-in and sign-up page. Please review my PR.
Live: https://musing-villani-3229e2.netlify.app/pages/authentication/authentication

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
looks good to me.

File: Pages/ProductPage/ProductPage.css [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 .filter-sidebar {
-    width: 25%;
+    width: 20%;
     height: 86vh;
     top: 8rem;
     overflow-y: scroll;
     position: sticky;
-    font-size: 1.6rem;
+    font-size: 1.3rem;
```
Comment (venkateshwarans) [inline Pages/ProductPage/ProductPage.css:9]: try to keep it in multiples of 5. ex - `1.25` or `1.5`

**Review by venkateshwarans — APPROVED** [review]
UI looks very pro. Nice choice of images.

File: Pages/ProductPage/ProductPage.css (hunk 1 above)
Comment (vrinda-mahajan) [reply Pages/ProductPage/ProductPage.css:9]: Okay👍

**Comment by b0llu** [comment]
The footer is not in the bottom for bigger screens. Everything else look good!

**Comment by vrinda-mahajan** [comment]
> The footer is not in the bottom for bigger screens. Everything else look good!

Thanks for pointing that out. I will correct it.

**Pull request merged**